charts/coms/values.yaml

```diff
@@ -185,13 +188,19 @@ objectStorageSecretOverride:
   # -- Object storage password
   password: ~
 
-# Patroni subchart configuration overrides
-patroni:
-  # -- Controls whether to enable managing a Patroni db dependency as a part of the helm release
-  enabled: false
-
-  # replicaCount: 3
-  # resources: {}
-
-  # persistentVolume:
-  #   enabled: false
+# crunchy-postgres subchart configuration overrides
+postgres:
+  enabled: true
+  # crunchy config:
+  fullnameOverride: ~
```

Comment:
the `nameOverride` value is the only one we need to override from COMS. we are setting it to `postgres`.
crunchydb templates create a `fullname` variable equal to `postgres-<release.name>` or the value of `fullnameOverride` (if provided) which is used for all the crunchyDB resources. eg: for the dev,test and prod (master) release, stateful sets are named: `postgres-master-ha-abc` and for a pull request in the dev environment it would be named:`postgres-pr-123-ha-abc`